js/js_test.go

```diff
@@ -467,7 +467,7 @@ func TestCallWithNull(t *testing.T) {
 func TestReflection(t *testing.T) {
 	o := js.Global.Call("eval", "({ answer: 42 })")
 	if reflect.ValueOf(o).Interface().(*js.Object) != o {
-		t.Fail()
+		t.Fatal()
```

Comment:
To be honest I was being a little lazy here in just flipping the existing `t.Fail()` to `t.Fatal()` to help with debugging step 2 from the TODO. 

I'm sure we could add helpful messages to the `t.Fatal()` calls... because the existing `t.Fail()` calls are equally lossy. Could even be something we do in another PR.